### PR TITLE
Update kernel instructions

### PR DESCRIPTION
**Pull request by robotdad**
I walked through the kernel instructions and have made some clean ups for clarification and added additional flags to support Arduino.

### CODE REVIEW

**Comment by benmcmorran** [comment]
@dorssel This contributor @robotdad is a PM that works with my team. I'm approving him for workflow runs.

File: WSL_USBIP.md [hunk 1]
```diff
@@ -110,43 +110,53 @@ uname -r
...
+You may need to set CONFIG_USB=y in .config prior to running menuconfig to get all options enabled for selection.
+
+Run menuconfig to select kernel features to add.
 
 ```bash
 sudo make menuconfig
 ```
 
-Select desired features. In my case I selected USB/IP, VHCI HCD, Debug messages for USB/IP, USB Serial Converter Support.
+These are the necessary additional features in munconfig.
```
Comment (benmcmorran) [inline WSL_USBIP.md:137]: These lines have some trailing whitespace that the Markdown linter is complaining about.

**Comment by benmcmorran** [comment]
Do you think it would be better to remove the `WSL_USBIP.md` file entirely and move the content to the project wiki instead? I don't have strong feelings either way, but right now documentation is split across both sources.

File: WSL_USBIP.md (hunk 1 above)
Comment (robotdad) [reply WSL_USBIP.md:137]: Interesting, the extra whitespace is to cause a line break which I thought was valid markdown.

**Comment by benmcmorran** [comment]
Sorry, I think my comment ended up on the wrong line. Blank lines between
paragraphs should be fine, but there's trailing whitespace at the ends of
lines with the menuconfig options.

**Review by dorssel — COMMENTED** [review]
Thx for the improvements.

After this is merged I will move this document to Wiki as @benmcmorran suggested and change the link in README. That makes it easier to update in the future. But... I don't exactly know how GitHub manages write access to Wiki. Can you check if you can edit any of the existing docs? For example, add the Arduino programmer to the list of working devices?

And don't worry about the CodeQL check failing. I think it was a one-off fluke. If not, I will modify the GitHub action to exclude the InstallerDependencies project (which is what it is complaining about).

**Pull request merged**